spring-cloud-dataflow-server-local/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+
+services:
+  kafka:
+    image: wurstmeister/kafka:0.10.1.0
+    expose:
+      - "9092"
+    environment:
+      - KAFKA_ADVERTISED_PORT=9092
+      - KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181
+    depends_on:
+      - zookeeper
+  zookeeper:
+    image: wurstmeister/zookeeper
+    expose:
+      - "2181"
+    environment:
+      - KAFKA_ADVERTISED_HOST_NAME=zookeeper
+  dataflow-server:
+    image: springcloud/spring-cloud-dataflow-server-local:latest
```

Comment:
For some reason, the deployed server is running on 1.3.1 BUILD-SNAPSHOT. 

![image](https://user-images.githubusercontent.com/1562654/37007697-6fd3ee80-2093-11e8-9ab0-441d2dc3427b.png)

Are we sure we are the `master` branch is configured to produced the image?

Reply:
I updated the CI plan for master to do the builds so unless I've done something wrong, it should be. I will ping you offline

Reply:
seems like a cached container - will look into something better but for the time being to update cache for `latest` changes: `docker-compose pull ; docker-compose up`